ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml

```diff
@@ -1521,30 +1521,23 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: baremetal-lab-ipi
-- as: baremetal-upi-ovn-ipv4-arm-mixarch-f14-day2-64k-pagesize
+- as: baremetal-ipi-ovn-ipv4-amd-vmedia-f7
```

Comment:
@jadhaj ^

as suggested, try to use two commits: one for the changes in step-registry/, the other for the test config, and the job created by the `update` Makefile target. It should ease the rebase.

Reply:
So actually these 2 files should be in one commit:
openshift-openshift-tests-private-release-4.16__multi-nightly.yaml
openshift-openshift-tests-private-release-4.16-periodics.yaml


and the rest in other one ?

Reply:
yes. Alternatively, you can keep one commit for step-registry (msg: `implement logic x`) , one for the test config (msg: `add test config for profile y`), and a third one for the generated job (msg: `add job for test config y`). This way, when you have to rebase, you could even drop the third commit with the job (which is slight harder to rebase as auto-generated), fix those few bits in the test config and run `make update` to generate a newer job (and third commit). 

The concept is: you should have a 'clear history of commits', useful for your and others use: squash temporary commits by grouping them into "functional/clever" clusters.

Reply:
Since commit history was squashed, so the way to do it is to soft reset and then push separated commits